apps/stats/stats-1.py

```diff
@@ -201,7 +201,13 @@ def __printVersion( self, script ) :
 		else :
 			version = "unknown"
 
-		print "Gaffer Version : {version}".format( version = version )
+		versions = (
+			( "From the script", version ),
+			( "Used for stats", Gaffer.About.versionString() ),
```

Comment:
Maybe this is nitpicking, but these labels seem a bit awkward/nonobvious. Not sure I have a better suggestion - maybe just "File" and "Current"?